plugins/tours/src/admin/wp-list-table.php

```diff
@@ -58,14 +58,15 @@ function _render_custom_column_content( $column_name, $tour_id ) {
 /**
  * Set sortable columns on the Tours admin page.
  *
- * @return array New sortable columns.
+ * @param array $sortable_columns An array of sortable columns.
+ * @return array  New sortable columns.
  * @since 1.0.0
  *
```

Comment:
`@since` should come before `@param` like this:

```php
/**
 * Set sortable columns on the Tours admin page.
 *
 * @since 1.0.0
 *
 * @param array $sortable_columns An array of sortable columns.
 * @return array New sortable columns.
 */
```

Reply:
@hellofromtonya Updated.